src/main/java/com/powsybl/openloadflow/ac/DistributedSlackOnLoadOuterLoop.java

```diff
@@ -24,10 +26,15 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(DistributedSlackOnLoadOuterLoop.class);
 
     private boolean distributedSlackOnConformLoad = false;
-
-    public DistributedSlackOnLoadOuterLoop(boolean throwsExceptionInCaseOfFailure, boolean distributedSlackOnConformLoad) {
-        super(throwsExceptionInCaseOfFailure);
-        this.distributedSlackOnConformLoad = distributedSlackOnConformLoad;
+    private LoadFlowParameters loadFlowParameters;
+    private OpenLoadFlowParameters openLoadFlowParameters;
+
+    public DistributedSlackOnLoadOuterLoop(LoadFlowParameters loadFlowParameters,
```

Comment:
Can you here have as arguments only the parameters you need for this outerloop ?

Reply:
From my mind, it's better to pass the two parameters objects instead of multiple primitive types ?
Moreover this code is done only in DistributedSlackOnLoadOuterLoop constructor (not in callers) :
`this.distributedSlackOnConformLoad = loadFlowParameters.getBalanceType() == LoadFlowParameters.BalanceType.PROPORTIONAL_TO_CONFORM_LOAD;`

So keep or not ?